minidump-stackwalk/README.rst

```diff
@@ -70,5 +70,40 @@ Building with Taskcluster
 The ``build-stackwalker.sh`` script will download a pre-built breakpad
 client binary if possible.
 
-To update this binary, you should use the ``scripts/breakpad-taskcluster.sh``
-script. Follow the directions in the header.
+To update this binary, copy the following task definition into the [TaskCluster
+task creation tool][task-creator]:
```

Comment:
This is a restructured text file, but I think this is Markdown markup.

Reply:
I thought I fixed this?

Oh guess I didn't push, whoopsie.